apps/nextjs/package.json

```diff
@@ -8,7 +8,8 @@
     "check": "tsc --noEmit",
     "clean": "rm -rf .next .turbo node_modules",
     "dev": "concurrently \"pnpm dev-server\" \"node scripts/local-dev.mjs\"",
-    "dev-server": "pnpm with-env next dev --port 2525 | pino-pretty -C",
+    "dev-turbo": "pnpm with-env node scripts/increase-listeners.js next dev --port 2525 --turbo | pino-pretty -C",
```

Comment:
Add a separate option for turbopack. Also increase the listeners to remove an error where we have too many